postmaster/apiv1/admins.py

```diff
@@ -147,3 +149,112 @@ def unlock_admin(admin_id):
     admin = Admins.query.get_or_404(admin_id)
     clear_lockout_fields_on_user(admin.username)
     return {}, 200
+
+
+@apiv1.route('/admins/<int:admin_id>/2factor', methods=['GET'])
+@login_required
+def twofactor_status(admin_id):
```

Comment:
I'd rather not have this route and just have the user of the API query the admin itself.